docs/commands.md

```diff
@@ -23,8 +23,9 @@ Commands:
   annotate     Annotate reads in a BAM file with segments from the model.
   demultiplex  Separate reads into files based on which model they fit best.
   extract      Extract coding segments from the reads in the given bam.
-  filter       Filter reads by whether they conform to expected segment order.
+  filter       Filter reads by whether they conform to expected segment...
```

Comment:
Truncation in the documentation string here

Reply:
Copy / paste error.  Fixed!